trio/_dtls.py

```diff
@@ -331,12 +350,20 @@ class OpaqueHandshakeMessage:
     record: Record
 
 
+# Needs Union until <3.10 is dropped
```

Comment:
FWIW I don't think this comment is necessary, as long as we remember to run `pyupgrade` (or manually ctrl+f for unions) we'll manage. I guess this is up to you though.

Reply:
There's another comment like this in another file (forgot its name and forgot to comment when I saw it), so above applies to that too.

Reply:
that one spawned enough confusion (mostly in me probably) that I ended up writing a proper comment, so I don't think leaving that one hurts.